### PR TITLE
address pr

### PR DESCRIPTION
**Pull request by Parvez098**

### CODE REVIEW

File: routes/users.js [hunk 1]
```diff
@@ -90,5 +90,23 @@ router.get("/list/:page", (req, res) => {
     });
 });
 
+router.post("/address", authentication.validateToken, (req, res) => {
+    let user_id = req.obj.user_id;
+    let address = req.body.address;
```
Comment (arun-kumar-codes) [inline routes/users.js]: validate data using express-validator

File: middleware/authentication.js [hunk 2]
```diff
@@ -1,5 +1,8 @@
 const db = require("../model/user");
 const AccessToken = require("../model/access-token");
+const { check, validationResult } = require('express-validator/check');
+const { sanitizeBody } = require('express-validator/filter');
```
Comment (arun-kumar-codes) [inline middleware/authentication.js]: what is use of this

Comment (Parvez098) [reply middleware/authentication.js]: i did not use it in my code,

Comment (Parvez098) [reply middleware/authentication.js]: sir i have removed this library

**Pull request merged**